### PR TITLE
operator bpfman-operator (0.5.2)

### PR DESCRIPTION
**Pull request by anfredette**
The bpfman-operator v0.5.2 release is a patch release that adds support for
ppc64le and s390x, and contains several minor internal updates.

Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [ ] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [ ] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [ ] Have you tested your Operator with all Custom Resource Definitions?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [ ] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [ ] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Updates to existing Operators

* [ ] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [ ] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [ ] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [ ] Have you tested an update to your Operator when deployed via OLM?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [ ] Modify more than one operator
* [ ] Modify an Operator you don't own
* [ ] Rename an operator - please remove and add with a different name instead
* [ ] Modify any files outside the above mentioned folders
* [ ] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [ ] Description about the managed Application and where to find more information
2. [ ] Features and capabilities of your Operator and how to use it
3. [ ] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [ ] Human readable name and 1-liner description about your Operator
* [ ] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [ ] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [ ] Links to the maintainer, source code and documentation
* [ ] Example templates for all Custom Resource Definitions intended to be used
* [ ] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Review by msherif1234 — APPROVED** [review]
/LGTM

File: operators/bpfman-operator/0.5.2/manifests/bpfman-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,1156 @@
...
+              "primarynodeinterface": true
+            },
+            "nodeselector": {},
+            "priority": 0
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: OpenShift Optional
+    containerImage: quay.io/bpfman/bpfman-operator:v0.0.0
```
Comment (msherif1234) [inline operators/bpfman-operator/0.5.2/manifests/bpfman-operator.clusterserviceversion.yaml:298]: `v0.5.2`

File: operators/bpfman-operator/0.5.2/manifests/bpfman-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,1156 @@
...
+              - args:
+                - --health-probe-bind-address=:8175
+                - --metrics-bind-address=127.0.0.1:8174
+                - --leader-elect
+                command:
+                - /bpfman-operator
+                env:
+                - name: GO_LOG
+                  value: debug
+                image: quay.io/bpfman/bpfman-operator:latest
```
Comment (msherif1234) [inline operators/bpfman-operator/0.5.2/manifests/bpfman-operator.clusterserviceversion.yaml:1066]: `v0.5.2`

File: operators/bpfman-operator/0.5.2/manifests/bpfman-config_v1_configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+data:
+  bpfman.agent.healthprobe.addr: :8175
+  bpfman.agent.image: quay.io/bpfman/bpfman-agent:latest
+  bpfman.agent.log.level: info
+  bpfman.agent.metric.addr: 127.0.0.1:8174
+  bpfman.image: quay.io/bpfman/bpfman:latest
```
[the remark this answers is not in the document]
Comment (anfredette) [reply operators/bpfman-operator/0.5.2/manifests/bpfman-config_v1_configmap.yaml]: changed to v0.5.2

File: operators/bpfman-operator/0.5.2/manifests/bpfman-config_v1_configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+data:
+  bpfman.agent.healthprobe.addr: :8175
+  bpfman.agent.image: quay.io/bpfman/bpfman-agent:latest
```
[the remark this answers is not in the document]
Comment (anfredette) [reply operators/bpfman-operator/0.5.2/manifests/bpfman-config_v1_configmap.yaml]: changed to v0.5.2

File: operators/bpfman-operator/0.5.2/manifests/bpfman-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (anfredette) [reply operators/bpfman-operator/0.5.2/manifests/bpfman-operator.clusterserviceversion.yaml]: Done.

File: operators/bpfman-operator/0.5.2/manifests/bpfman-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (anfredette) [reply operators/bpfman-operator/0.5.2/manifests/bpfman-operator.clusterserviceversion.yaml]: Done

**Pull request merged**